master/internal/db/setup.go

```diff
@@ -49,3 +52,47 @@ func Setup(opts *config.DBConfig) (*PgDB, error) {
 	}
 	return db, nil
 }
+
+// ValidateRPWorkspaceBindings checks if rp-workspace bindings pertain to valid resource pools.
+// Bindings with resource pools that don't exist have the "valid" column set to false.
+func ValidateRPWorkspaceBindings(pools []config.ResourcePoolConfig) error {
+	var poolNames []string
+	for _, pool := range pools {
+		poolNames = append(poolNames, pool.PoolName)
+	}
+
+	ctx := context.Background()
```

Comment:
maybe `context.TODO()`, or take it as an arg for now. so it is less forgettable if we give `run` a context.